app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -107,6 +107,10 @@ def machine_id(feed)
       os_resource_for(feed).try(:properties).try { |prop| prop['Machine Id'] }
     end
 
+    def container_id(feed)
```

Comment:
The `machine_id` method is already here so if we are going to move it, best to do in a follow-up PR